app/src/main/java/org/schabi/newpipe/RouterActivity.java

```diff
@@ -127,8 +128,10 @@ protected void onCreate(final Bundle savedInstanceState) {
             }
         }
 
+        ThemeHelper.setDayNightMode(this);
         setTheme(ThemeHelper.isLightThemeSelected(this)
                 ? R.style.RouterActivityThemeLight : R.style.RouterActivityThemeDark);
+        assureCorrectAppLanguage(this);
```

Comment:
just a small thing, but I prefer to not import static methods but call them in a static way: Localization.assureCorrectAppLanguage(this);

this way it is directly clear that you are calling a static method of an utility class instead of a local one

Reply:
doen